ush/config.sh.rrfs_b_c3

```diff
@@ -20,7 +20,7 @@ DO_POST_SPINUP="FALSE"
 DO_POST_PROD="TRUE"
 DO_RETRO="FALSE"
 DO_NONVAR_CLDANAL="TRUE"
-DO_ENVAR_RADAR_REF="TRUE"
+DO_ENVAR_RADAR_REF="FALSE"
```

Comment:
Why is this turned off?

Reply:
We do not have RRFSE and this will not run. Just try to avoid confusion.

Reply:
Okay, sounds good! Thanks for the explanation!